USER_MANUAL.md

```diff
@@ -336,6 +363,345 @@ _TODO_
+
+**Notes on acquisition:** While the "true" LPR is elusive, it can be estimated by dividing the number of leaks found during an LDAR survey at a facility by the number of days that have passed since the previous LDAR survey at the same facility. If this is done for a large number of survey intervals at a large number of facilities, one should eventually converge on a representative estimate. When LDAR-Sim is used, operator-specific LPR values should be estimated if sufficient data exist to do so.
+
+**Notes of caution:**  Available techniques for estimating LPR make a number of problematic assumptions. Ultimately, we have relatively poor data on LPR and the relationship between LPR and NRd. Modeling results are extremely sensitive to LPR. Given that LPR is elusive, we strongly recommend that a broad range of LPR values is evaluated in LDAR-Sim before any decisions are made. For more information, refer to discussions in the main text and supplementary information of Fox et al. (2021).
+
+When the parameter [multi](#multi) is set to false, it will affect the emission production rate observed in the simulation. In this scenario, new emissions won't be generated if there's already an existing emission for the specified emission source.
+
+#### &lt;NRd&gt;
+
+**Data Type:** Integer (Numeric)
+
+**Default input:** 365
+
+**Description:** The natural repair duration or day(NRd) of each repairable emission in number of days. Represents emission removal from the repairable emission pool due to routine maintenance, refits, retrofits, and other unintentional emission repairs.
+
+**Notes on acquisition:** Estimate from empirical data or use previously published value.
+
+**Notes of caution:** This value is highly uncertain and likely depends on context. Sensitivity analyses should be used to explore the impact of different NRd values.
+
+The NRd value should be the same for **all** programs
+
+#### &lt;multi&gt;
+
+**Data Type:** Boolean
+
+**Default input:** True
+
+**Description:** Specifies whether a repairable emission source can generate multiple emissions simultaneously. For instance, a flare that's unlit can't produce additional unlit emissions simultaneously.
+
+**Notes on acquisition:** Users are encouraged to undertake an exercise to assess whether it's logical for a particular source to generate multiple emissions simultaneously.
+
+**Notes of caution:** When set to false, it's possible to observe lower emission production rates than what has been parameterized, as existing emissions inhibit the generation of new emissions.
+
+#### &lt;NR_ERS&gt;
+
+**Data Type:** String
+
+**Default input:** The name of a column in the [emissions file](#emissions_file) that is used to parameterize the emissions characteristics of all the general non-repairable emissions. This parameter will be overwritten by the equivalent value, when provided at a more granular scale.
```

Comment:
See my other comments about clarity with propagating parameters - Will need to update all propagating emissions parameters to be inline with each other.